.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+    - 2.7
+    - 3.8
```

Comment:
No. Please look at other Travis configs like https://github.com/zopefoundation/Zope/blob/4.x/.travis.yml and do it that way (except the email notification)